microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add ariaLabels on the Sample Queries documentation icons

## Overview

Fixes #2235 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Accessibility Insights Tool for web browsers needs to be installed on your browser for testing.

## Testing Instructions

* Checkout this branch
* Load Graph Explorer
* Run Fast Pass from the Accessibility Insights Tool on the browser
* Observe fix
